### PR TITLE
Update: remove need for tabs permission

### PR DESCRIPTION
**Pull request by PikaDude**
So I'm an idiot and thought we needed the "tabs" permission to check if tabs were playing audio. Nope I'm just a complete fool, we don't. At all. This PR completely removes it and changes the default tab audio behaviour option from "nothing" to "pause".

### CODE REVIEW

**Comment by jordigarcl** [comment]
WTF. I also reviewed the `chrome.tabs` docs for that PR and missed it as well. It's right there...
> The majority of the chrome.tabs API can be used without declaring any permission. However, the "tabs" permission is required in order to populate the url, pendingUrl, title, and favIconUrl properties of Tab.

I just checked the Wayback Machine in case that was a recent clarification but no, it's always been there 🤡🤣

**Comment by PikaDude** [comment]
I swear I got a permission error too when I tried it without the perms, but I guess I was doing something different back then.

File: options.html [hunk 1]
```diff
@@ -302,14 +304,12 @@ <h3>Version 4.0.0<span class="changelog-date"> - 8th of February 2020</span></h3
...
-			<a target="_blank" href="https://developer.chrome.com/extensions/permission_warnings#view_warnings">For more info, refer to the Chrome Developer Docs.</a>
+			<p id="modal-title"></p>
+			<p id="modal-text"></p>
+			<a id="modal-link" target="_blank"></a>
 			<br />
 			<br />
-			<button id="tabAudioModalDismiss">Understood!</button>
+			<button id="tabAudioModalDismiss"></button>
 		</div>
 	</div>
```
Comment (jordigarcl) [inline options.html:322]: Seeing as this is not used anywhere else, shouldn't we remove it completely?

Comment (PikaDude) [reply options.html:322]: Perhaps, but I'd say we keep it so we can have custom popups if we ever need them for anything else. No real harm tbh.

**Review by jordigarcl — APPROVED** [review]
Tested and LGTM. 
Agree that the default should be 'paused', as that's the less invasive yet sensible option.

File: options.html (hunk 1 above)
Comment (jordigarcl) [reply options.html:322]: No harm at all, but I'd still suggest to remove it so as not to confuse potential contributors. We can always go back to it through Git.

Comment (PikaDude) [reply options.html:322]: Eh, I don't think it'd confuse anyone. It's just a template that's unused.

**Pull request merged**